Resources/Prototypes/Entities/Structures/Doors/Airlocks/airlocks.yml

```diff
@@ -138,7 +136,10 @@
     group: Glass
   - type: RadiationBlocker
     resistance: 2
-
+  - type: Tag
+    tags:
+      - GlassAirlock
```

Comment:
Why did this need to be added?

Reply:
I think I understand. I think we should explain somewhere that the airlocks use tags so that the glass and non-glass versions can share the construction graphs.

Reply:
That's correct. And done.